{{cookiecutter.project_slug}}/Jenkinsfile

```diff
@@ -4,29 +4,41 @@ pipeline {
         stage("Checkout") {
             steps {
                 checkout scm
+                sh "mkdir -p var/static var/media var/log"
+                sh "rm -rf .venv"
                 sh "echo 'COMPOSE_PROJECT_NAME=${env.JOB_NAME}-${env.BUILD_ID}' > .env"
+                sh "echo 'SSH_AUTH_SOCK=/tmp/fake_ssh_socket' >> .env"
```

Comment:
Hé! Deze ken ik niet. Wat doet dit?

Reply:
Nou de docker-compose forward de SSH_AUTH_SOCK, wat handig is om binnen je docker private git repos uit te checken als de host daar een geldige SSH key voor heeft. Op Jenkins is dit helaas niet geconfigureerd, maar de docker-compose loopt stuk op een lege SSH_AUTH_SOCK. Dus we zetten er een fake_ssh_socket in.